src/programs/applyctf/applyctf.cpp

```diff
@@ -165,9 +193,19 @@ bool ApplyCTFApp::DoCalculation()
 			}
 		}
 
-		if (phase_flip_only == true) current_image.ApplyCTFPhaseFlip(current_ctf);
-		else current_image.ApplyCTF(current_ctf);
+		if (phase_flip_only == true) {current_image.ApplyCTFPhaseFlip(current_ctf);}
+		else if (apply_wiener_filter == true) {current_image.OptimalFilterWarp(current_ctf,
+											   pixel_size,
+											   wiener_filter_falloff_fudge_factor,
+											   wiener_filter_scale_fudge_factor,
+											   wiener_filter_falloff_frequency,
+											   wiener_filter_high_pass_radius);}
+		else {current_image.ApplyCTF(current_ctf);}
+		//current_image.ZeroCentralPixel();
 		current_image.BackwardFFT();
+		if (apply_wiener_filter == false) {
+			current_image.InvertRealValues();
+		}
 		current_image.WriteSlice(&output_file,image_counter + 1 );
```

Comment:
How come you are inverting here?

Reply:
Right, but that is the correct behavior after applying the CTF.

Maybe add an option "keep original image contrast?" or something like that.

Reply:
Sure, that sounds reasonable.

Reply:
Ok, I added the option. The behavior is now to invert the contrast by default but maintain the image contrast when setting maintain_image_contrast to Yes
![image](https://user-images.githubusercontent.com/6081039/128539604-ca2c9b75-65d8-4ed0-ad1b-d78456434475.png)

Reply:
The program is called applyctf. Running this program using default parameters, I expect that CTF, not (-1.0)*CTF, is applied.
An extra contrast inversion should not be the default in my opinion.

Reply:
Sorry - I re-read your message. I now realize that this is what you implemented already :)